lib/Dialect/Torch/Transforms/AdjustCallingConventions.cpp

```diff
@@ -187,8 +187,47 @@ class AdjustCallingConventionForReturn
 };
 } // namespace
 
+static bool isValidNonContainerResultType(Type resultType) {
+  return resultType.isa<Torch::BaseTensorType>() ||
+         resultType.isa<Torch::FloatType>() ||
+         resultType.isa<Torch::IntType>() ||
+         resultType.isa<Torch::BoolType>() ||
+         resultType.isa<Torch::NoneType>();
+}
+
+static LogicalResult validateReturns(func::FuncOp func) {
+  if (func.getResultTypes().size() > 1) {
+    return func->emitError(
+      "Functions can only ever return one item. Multiple return values are "
```

Comment:
This is not really precise. I would phrase this as "Functions directly imported from Python should..."

Reply:
Updated.  LMK what you think of the new phrasing.  I also added a comment above the part where we return if nothing has been returned (as distinct from `None`).  Let me know if there's a better way to phrase that comment.